### PR TITLE
[Feature #162727528] Bookmark and List Articles

### PR DESCRIPTION
**Pull request by ebenezerdon**
#### What does this PR do?
This PR enables a user to:
- bookmark an article
- remove a bookmark
- view bookmarked articles

#### Description of Task to be completed?
- Users should be able to bookmark articles for reading later via a bookmark tab, button or link
- An authenticated user should be able to see all his bookmarked articles in his/her profile and navigate to them

#### How should this be manually tested?
- navigate to `/article/:slug`
- When the `bookmark` icon is empty, it indicates to the user that a particular article has not yet been bookmarked.
 - - Click on the  icon to add it to your bookmarks

- When the `bookmark` icon is filled, it indicates to the user that a particular article has been bookmarked.
- -  Click on the icon to remove it from your bookmarks

#### What are the relevant pivotal tracker stories?
- [#162727528](https://www.pivotaltracker.com/story/show/162727528)
- [#162727529](https://www.pivotaltracker.com/story/show/162727529)

### CODE REVIEW

File: src/app/article/ArticleComponent.jsx [hunk 1]
```diff
@@ -131,7 +192,11 @@ class ArticleComponent extends React.Component {
               />
             </div>
             <div className="left-sidebar-down">
-              <div className="side-bookmark" />
+              {/* <Link
```
Comment (andela-mabdussalam) [inline src/app/article/ArticleComponent.jsx]: Why is this commented?

Comment (ebenezerdon) [reply src/app/article/ArticleComponent.jsx]: I've resolved this.

File: src/app/article/ArticleComponent.jsx [hunk 2]
```diff
@@ -21,10 +28,64 @@ class ArticleComponent extends React.Component {
...
+            id={removeBookmarkId}
+            type="button"
+            onClick={() => doRemoveBookmark(data.slug)}
+          >
+            <i className="material-icons">bookmark</i>
+          </button>
+        </>
+      );
+    }
+    if (
```
Comment (andela-mabdussalam) [inline src/app/article/ArticleComponent.jsx]: Is this if statement necessary?

Comment (ebenezerdon) [reply src/app/article/ArticleComponent.jsx]: I've removed it

**Review by andela-mabdussalam — APPROVED** [review]

**Pull request merged**